polygeist/tools/cgeist/Test/Verification/compoundassign.c

```diff
@@ -171,3 +173,64 @@ unsigned f11(unsigned *a, unsigned b) {
 unsigned f12(unsigned *a, unsigned b) {
   return *a ^= b;
 }
+
+// COM: This test is incorrect due to signedness detection issues.
```

Comment:
I would prefer to have a test that XFAILS and has the correct code  ... but this is up to you.